src/index.js

```diff
@@ -46,47 +44,35 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z0-9]*_[A-Z0-9]+$'],
```

Comment:
We seem to be requiring it in instances such as `this.__class__ = 'Foo';`. Or is that a property and not an id? We may also need something like this for `const _ = require('lodash');`.

Anyway, as a rule of thumb, I'm up for adapting rules to our needs and not simply blindly porting it.